ext/mysqli/mysqli.stub.php

```diff
@@ -4,14 +4,11 @@
 
 final class mysqli_driver
 {
-    /** @var string|null */
-    public $client_info;
+    public string $client_info;
 
-    /** @var int|null */
-    public $client_version;
+    public int $client_version;
 
-    /** @var int|null */
-    public $driver_version;
+    public int $driver_version;
 
     /** @var bool|null */
     public $reconnect;
```

Comment:
What about this guy?

Reply:
I think these are type hinted as nullable because they are untyped properties (so their implicit default value is `null` rather than uninitialized) and doesn't have an explicit non-nullable default value. So far we followed this rule of thumb when deciding about the type of properties. However, in https://github.com/php/php-src/pull/6742#discussion_r584556169 I proposed hat we should probably omit null from the documentation.

Reply:
Not sure I follow. `reconnect` is a boolean with the default value false. `report_mode` is a bitmask with default value 0. Neither of them can ever be null. Is there something I am missing?

Reply:
You only miss that I haven't known that they have a default value which can be added to the stubs :) It's fixed now!